### DISCUSSION TITLE
Hw3 Part1 Q1

### DISCUSSION THREAD

**Discussion by HuyNguyen1111** [cooperrc/computational-mechanics#106] [Q&A]
![image](https://user-images.githubusercontent.com/112562702/193331967-3562cfee-7d0d-4e54-bbc8-bcea5193c0dc.png)

![image](https://user-images.githubusercontent.com/112562702/193331806-19e87f17-d8b4-4d34-8c8e-49d98348e227.png)

I've done the notes section for part 1 but even after saving my work/coordinates it still doesn't load the file.

**Comment by RebeccaVillanueva** [comment]
I had to use '../data/projectile_coords.npz' instead of just 'projectile_coords.npz' for it to work

**Reply by poutrej** [reply]
Mine still doesn't load even using the whole phrase like you did
